components/home/ClassroomSubjectCard.tsx

```diff
@@ -32,6 +33,18 @@ const ClassroomSubjectCard: StylableFC<{
   const locale = useLocale();
   const { t } = useTranslation("schedule", { keyPrefix: "subjectList.card" });
 
+  const mysk = useMySKClient();
+
+  /** If applicable, ensure the URL is to the right account. */
+  const url = subject.ggc_link
+    ? mysk.user?.email
+      ? `https://accounts.google.com/AccountChooser?${new URLSearchParams({
```

Comment:
Having a normal Google Classroom link as a fallback is a good catch if there's an error.